lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1426,6 +1431,42 @@ newtype ApiMnemonicT (sizes :: [Nat]) =
     ApiMnemonicT { getApiMnemonicT :: SomeMnemonic }
     deriving (Generic, Show, Eq)
 
+
+data ApiOurStakeKey n = ApiOurStakeKey
+     { _index :: ApiT DerivationIndex
```

Comment:
Seems `DerivationIndex` is encoded as a JSON string, which isn't what I want. Going with `Natural` then, I guess. Could maybe have some `ApiSoftIndex` newtype if desired…